source/examples/xgboost-randomforest-gpu-hpo-dask/notebook.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>Similar to above, the URL should be https://ml.dask.org/modules/generated/dask_ml.model_selection.RandomizedSearchCV.html above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>